src/patterns/asking-for-feedback/index.md.njk

```diff
@@ -0,0 +1,146 @@
+
+Where it would be useful to know who someone is, or to collect other personally identifying information you need to be clear about:
+
+- why you’re collecting it
+- what you’re collecting
+- how you'll use it
+- who has access to it
+- how long we’ll keep it
+- that it will be secure
+- how they can find out more or raise a concern or complaint
+
+[Find out more about the Data Protection Act and what it covers](https://www.gov.uk/data-protection)
+
+## Welsh language services
+
+Services in Welsh should provide feedback forms in Welsh.
+
+Check our [HM Land Registry Welsh Language Scheme](https://www.gov.uk/government/organisations/land-registry/about/welsh-language-scheme) if you are not sure if this applies to your service.
+
+If your service is being used by Welsh speaking people, you should do this as soon as possible. By public beta all parts of the service should be in Welsh including the feedback form.
+
+Follow our guidance for help with developing a Welsh version of your form.
+
+Contact Search for land and property information for information team to find out how they transformed their feedback form to get more useful results from their users.
+
+## Related links
+
+- [Get a feedback page on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-feedback-page)
+- [How to structure forms](https://www.gov.uk/service-manual/design/form-structure)
+
+## Why you should follow this pattern advice
+
+It’s also part of your service under the law. For example, in the Equality Act 2010 we fall under the 'service provider' category.
+
+The rest of the team should consider this page as they would any other in the services, for example:
+
+- lawyers confirming compliance with all laws
+- testers ensuring its technical accessibility
+- software developers should ensure, for example it’s secure, is available, what happens if the service (or part of) is not available.
+- user researchers and designers to ensure its usable and accessible (non-technical compliance
+
+The feedback form must be accessibility tested as part of the accessibility statement.
+
+## Accessibility and inclusivity
+
+Everything we provide in the operation of a service must be accessible and not discriminate.  We are the ‘service provider’ in the Equality Act 2010, where it is illegal to discriminate against people with certain characteristics, for example race and disability.
+
+This applies to internal services as well as public and professional facing services.
+
+This applies to people using both the digital and non-digital parts of your service.
+
+How to:
+
+- [Make your service Inclusive (non-discriminating)](https://www.gov.uk/service-manual/design/making-your-service-more-inclusive)
+- [Provide accessible services and assisted digital support](https://www.gov.uk/service-manual/helping-people-to-use-your-service)
+
+Does a service have to do it? (Welsh – think this lives n the Welsh guidance)
```

Comment:
remove the following content from this page

Does a service have to do it? (Welsh – think this lives n the Welsh guidance)
Dunno - to write. Ask a lawyer, ask them early on - e.g., early in Alpha."